### PR TITLE
Add Color::FromUInt.

### LINKED ISSUE #2179: Add Color::FromUInt(unsigned int);
There exists a Color::ToUInt() it would be nice if there was a Color::FromUInt(unsigned int).  To avoid having to unpack picks bits in outside code.

### PR DESCRIPTION
**Pull request by hsnabn**
Closes #2179.

This is my first contribution of substance to any open source project, so please, leave any comments on how to improve.

### CODE REVIEW

**Comment by hsnabn** [comment]
Ah, it seems I should've checked the active pull requests before starting to work. #2183 already accomplishes this.

However, in my pull, I added the new function to the Lua and AngelScript APIs as well (which I assume are needed for them to be used in script).

Or should I close my pull?

File: Source/Urho3D/Math/Color.cpp [hunk 1]
```diff
@@ -64,6 +64,14 @@ Vector3 Color::ToHSV() const
     return Vector3(h, s, v);
 }
 
+void Color::FromUInt(unsigned color)
+{
+    a_ = (float)(((color >> 24) & 0xff) / 255);
```
Comment (1vanK) [inline Source/Urho3D/Math/Color.cpp]: color / 255 int diivision?

Comment (hsnabn) [reply Source/Urho3D/Math/Color.cpp]: Should I be converting 255 to unsigned? Or something else?

Comment (1vanK) [reply Source/Urho3D/Math/Color.cpp]: when you divide two integer numbers, result is rounded to int
just write 255.0f

Comment (hsnabn) [reply Source/Urho3D/Math/Color.cpp]: Done in last commit.

**Comment by eugeneko** [comment]
I merged this PR just because it was simpler this time.
Please notify community if you are implementing or fixing reported issues.

**Comment by hsnabn** [comment]
@eugeneko Just for future reference, how do you mean by 'notify community'? Discuss on the forums?

**Comment by eugeneko** [comment]
If it's something wide-scale, yes. It's better to discuss on forums.
If it's something small, then just a message from the issue is enough.

**Pull request merged**